Lesson24/selcan1/1.html

```diff
@@ -0,0 +1,127 @@
+            background-color: rgb(60, 60, 104); 
+            display: flex; 
+            justify-content: center;  
+            align-items: center;    }
+        .anaMenu{
+            border-radius: 25px;
+            background-color: white;
+            width: 50%;
+            height: 50%;
+            padding: 30px;
+            display: flex;
+            flex-direction: row;  
+         }
+         .sagEkran{
+            display: flex;
+            margin-left: 90px;
+            justify-content: center;
+         }
+        .sagEkran>img{
+            width: 90%;
+            height: 100%;
+            border-radius:20px;
+        }
+         h2{
+            font-size: 30px;
+            padding: 0 20px;
+         }
+         ul{
+            line-height: 30px;
+            padding-left: 0;
+         }
+         .solEkran li{
+            list-style-type: none;
+            padding-left:10px;  
+         }
+         ul .kutu{
+            display: flex;
+            align-items: flex-start;
+         }
+         .form-input{
+            width: 100%;
+            height: 35px;
+            border-radius: 5px;
+            margin-bottom: 20px;
+         }
+         .form-input,.form-label{
+            letter-spacing: 2px;
+            margin-top: 10px;
+         }
+         button{
+            width:calc(100% + 10px);
+            height: 50px;
+            background-color: rgb(4, 4, 30);
+            color: white;
+            padding: 5px auto;
+            border-radius: 10px;
+            cursor: pointer; 
+         }
+         button:hover{
+            background-color: #590808;
+            transition: .5s;
+         }
+         .error {
+            color: #590808;
+            font-size: 12px;
+            margin-top: 5px;
+            margin-bottom: 10px;
+        }
+
+        
+
+
+    </style>
+</head>
+<body>
+    <div class="kapsayici">
+        
+       <div class="anaMenu">
+        <div class="solEkran">
+            <h2>Stay Updated!</h2>
+            <p>Join 60.0004 product managers receiving monthly updates on:</p>
+           <ul>
+               <div class="kutu"> <input type="checkbox"><li>Product discovery and building what matters</li></div>  
+                <div class="kutu"><input type="checkbox"><li>Measuring to ensure updates are success </li></div>
+                <div class="kutu"><input type="checkbox"><li>And much more!</li></div>
+            </ul>
+            <form id="subscribeForm" novalidate>
+            <label for="email" class="form-label" >Email Address:</label>
+            <div id="emailError" class="error"></div>
+            <input type="email" id="email" name="email" class="form-input" placeholder="Email@company.com"required>
+            <div class="buton"><button>Subscribe to monthly newsletter</button></div></form>
+        </div>
+        <div class="sagEkran">
+            <img src="./1.png" alt="resim1" ">
+        </div>
+        </div>
+    
+    </div>
+    <script>
```

Comment:
Move JS code to an external file, same for CSS

Reply:
I make them in external files in normal projecs but it is easier to send all of them in one file,to be able to find them in one file and while wrighting the codes,it is coming easier to be able to see everything in one file.Until buying external screens,can I continue like tis?

Reply:
You can do it for now but even if you work with one screen, try using "Open to the side" option (in the menu when you right-click on the file) to open files side by side as keeping everything in the same file is not a good solution long term. Very soon we will start using React and will work with multiple files so it's good to get used to work easily with multiple files at the same time.